eralchemy/main.py

```diff
@@ -22,6 +22,20 @@
     metadata_to_intermediary,
 )
 
+USE_PYGRAPHVIZ = True
+try:
+    from pygraphviz.agraph import AGraph
+
+    logging.debug("using pygraphviz")
+except ImportError:
+    USE_PYGRAPHVIZ = False
+    try:
+        from graphviz import Source
+
+        logging.debug("using graphviz")
+    except ImportError:
+        logging.error("either pygraphviz or graphviz need to be installed")
```

Comment:
At that point both AGraph and Source are not defined but the below code assumes that one is. I would recommend to add another raise

Reply:
Well eralchemy works fine without graphviz/pygraphviz, if you do not want to output PDF, PNG, JPG, SVG directly..

I'll add another check GRAPHVIZ_AVAILABLE and raise on usage